src/audio/crossover/crossover.c

```diff
@@ -636,3 +636,22 @@ static const struct module_interface crossover_interface = {
 
 DECLARE_MODULE_ADAPTER(crossover_interface, crossover_uuid, crossover_tr);
 SOF_MODULE_INIT(crossover, sys_comp_module_crossover_interface_init);
+
+#if CONFIG_COMP_CROSSOVER_MODULE
+/* modular: llext dynamic link */
+
+#include <module/module/api_ver.h>
+#include <module/module/llext.h>
+#include <rimage/sof/user/manifest.h>
```

Comment:
the crossover depends on math iir. Is there any risk of breaking that here?

Reply:
@cujomalainey as long as iir math is built into the image - no, but if / when we make it modular too, then we'll need to make sure to build and load it to satisfy the dependency

Reply:
@cujomalainey next steps are to make maths library (and other utility code) into per feature "modules" and load into SRAM on demand i.e. minimize SRAM use for power and for active modules only, further along the road is using llext to put hot (time sensitive) code in SRAM and cold code in DRAM to further optimize/detereminize processing performance.